src/rust/ensogl/src/display/symbol/gpu/geometry/compound/sprite.rs

```diff
@@ -89,6 +89,16 @@ impl {
         self.display_object.set_position(value)
     }
 
+    /// Modifies the rotation of the sprite.
+    pub fn mod_rotation<F:FnOnce(&mut Vector3<f32>)>(&self, f:F) {
+        self.display_object.mod_rotation(f);
+    }
+
+    /// Sets the rotation of the sprite.
+    pub fn set_rotation(&self, value:Vector3<f32>) {
+        self.display_object.set_rotation(value)
+    }
+
```

Comment:
This is not needed anymore. `display::object` exports traits adding these methods to everything which is. display object. In fact we should remove some of the methods, like `position` as they are auto-implemented nowadays.